Bundler/Viewer/Unity/Assets/Scripts/Network/Scripts/NetworkAnchor.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using Persistence;
+
+public class NetworkAnchor : MonoBehaviour
+{
+    /// <summary>
+    /// The game object to enable when the initial network anchor is being loaded.
+    /// </summary>
+    public GameObject LoadingAnchorRoot;
```

Comment:
add [Tooltip("...":)] to publics so that hovering in the inspector shows the purpose of the field

Reply:
changed.  replaced summaries with tooltips